_schemaV2.graphql

```diff
@@ -2692,6 +2692,16 @@ interface ArtworkEdgeInterface {
   node: Artwork
 }
 
+# An error type, potentially containing a partial artwork response
+type ArtworkError {
+  artwork: PartialArtwork
+  errorDetail: ArtworkErrorDetail
+}
+
+type ArtworkErrorDetail {
+  statusCode: Int!
+}
```

Comment:
If the idea here is to elevate and apply this more generally, could we make this a bit more system centric? Ie, something like 

```graphql
type PageError = ArtworkPageError | ShowPageError | FooPageError

type PageErrorDetail {
  statusCode: Int
}
```

Thinking these types are a bit too generic, and should this pattern spread there will soon be a lot of unnecessary duplication.

Reply:
Yes, I also thought that the basic 'error' type here isn't artwork-specific and could be more general, but that the duplication wouldn't be _that_ bad  as it's a basic type (vs. duplicating more business-y schema concepts).

I think you're right and might as well introduce that abstraction to the error detail to start with.

I'll give that another pass.

Reply:
I rolled with `RequestError` for now.